src/UI/CardSelector.hs

```diff
@@ -57,7 +57,13 @@ drawUI s =
   [ drawMenu s <=> drawException s ]
 
 title :: Widget Name
-title = withAttr titleAttr $ hCenteredStrWrap "Select a deck of flashcards"
+title = withAttr titleAttr $ str "Select a deck of flashcards "
+
+shuffledWidget :: State -> Widget Name
+shuffledWidget s = withAttr exceptionAttr $ str $ 
```

Comment:
Instead of using the existing `exceptionAttr`, I would make a new attribute for the shuffling, like `shuffleAttr` or something. Doing it this way means we get a lot of attributes, but in the future if the `exceptionAttr` is made bold for example it's not desirable that the shuffle text also becomes bold.